addons/web_editor/static/src/js/editor/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -98,8 +98,8 @@ describe('Copy', () => {
                     const clipboardData = new DataTransfer();
                     triggerEvent(editor.editable, 'copy', { clipboardData });
                     window.chai.expect(clipboardData.getData('text/plain')).to.be.equal('First');
-                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('<li>First</li>');
-                    window.chai.expect(clipboardData.getData('text/odoo-editor')).to.be.equal('<li>First</li>');
+                    window.chai.expect(clipboardData.getData('text/html')).to.be.equal('First');
```

Comment:
why do we care of copying a single `LI` when we are about to unwrap it in [insert](https://github.com/odoo/odoo/blob/16.0/addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js#L182C1-L191C56).

Reply:
If the `ul` element is also wrapped during copying (which occurs when multiple items of the list are copied), then upon pasting, it will have a bullet point. However, it has been decided not to wrap the `ul` element when only a single list item is copied in this [PR](https://github.com/odoo/odoo/pull/119967#issuecomment-1808676355).

Reply:
Thanks that's clear then!